libs/api/domains/national-registry/src/lib/soap/dto/getViewFjolskyldanDto.ts

```diff
@@ -0,0 +1,113 @@
+export interface GetViewFjolskyldanDto {
```

Comment:
GetViewFamilyDto ?

Reply:
I know this is an icelandic soap api but to what extent can our code be in English? Maybe it doesn't make sense to do it, I just don't know where the line is

Reply:
Yes well im not sure either. Just wanted the NationalRegistryApi to handle the icelandic mambo jambo.